src/components/CoursesSidebar/CoursesSidebar.jsx

```diff
@@ -18,10 +16,11 @@ export default function CoursesSidebar({ isAdmin }) {
   return (
     <nav className={navigationStyles.navigation}>
       <div className={navigationStyles.navButton}>
-        <ButtonEmptyBorder
-          type="button"
+        <Button
+          HTMLType="button"
+          type="emptyBorder"
```

Comment:
```suggestion
          type="button"
          mode="emptyBorder"
```
лучше делать так, чтобы нативные хтмл атрибуты оставались такими же, а свои назывались как-нибудь по-другому. Нативные атрибуты они у всех девелоперов на всех проектах одинаковые, поэтому когда видишь `<Button type>` то в первую очередь думаешь, что это нативный атрибут